TASK-5/Customer.cs

```diff
@@ -0,0 +1,32 @@
+namespace Task5;
+
+public class Customer
+{
+    private string _passportId;
+    private string _name;
+    private int _age;
+    private Cart _cart;
+
+    public string PassportId { get => _passportId; set => _passportId = value; }
+    public string Name { get => _name; set => _name = value; } 
+    public int Age { get => _age; set => _age = value; }
+    public Cart Cart { get => _cart; set => _cart = value; }
+
+    public void SetCustomer(string passportId, string name, int age, Cart cart)
+    {
+        PassportId = passportId;
+        Name = name;
+        Age = age;
+        Cart = cart;
+    }
+
+    public bool Equals(Customer customer)
+    {
+        return PassportId == customer.PassportId;
+    }
```

Comment:
For this realization override Object.Equals method. Because you break the naming convention rule

Reply:
I've overridden the method, still I don't get what's wrong with the naming after having surfed the Google. Could you give the clue please?